src/Sulu/Bundle/MediaBundle/Admin/MediaAdmin.php

```diff
@@ -134,7 +134,6 @@ public function configureRoutes(RouteCollection $routeCollection): void
                     ->setResourceKey('media')
                     ->setFormKey('media_details')
                     ->setTabTitle('sulu_media.information_taxonomy')
-                    ->setEditRoute(static::EDIT_FORM_DETAILS_ROUTE)
```

Comment:
This route was directing to itself here. Wondering if we should catch that in the `RouteBuilder` and throw an exception, to make clear that this does not work correctly.

Reply:
Throwing a LogicException or something similiar sounds like a good idea for me to avoid bug like this.